src/pages/starter-kit/checkout/getting-started.md

```diff
@@ -0,0 +1,32 @@
+---
+title: Getting started with checkout starter kit
+description: Learn about the Adobe Commerce checkout starter kit and how you can get started.
+keywords:
+  - App Builder
+  - Extensibility
+---
+
+# Getting started
```

Comment:
I believe the sections `Development > Prerequisites` and `Development > Initial configuration` should be part of this getting started page since they're not part of the development of the app but part of how to bootstrap the starter kit locally.

Reply:
When `Initial configuration` is moved here, there's a missing step in there which would be something like

> 4. Copy the env variables from `env.dist` in the root of the project to `.env` and fill in the required values.
>
> After this you should be ready to deploy the initial project in app builder, but first you might want to check the scripts provider in order to configure the Commerce events provider and its subscriptions, any 3rd party events or an out of process payment method.